source/dyaml/test/reader.d

```diff
@@ -38,13 +38,13 @@ void runReader(const bool verbose, void[] fileData)
 ///
 /// Params:  verbose       = Print verbose output?
 ///          errorFilename = File name to read from.
-void testStreamError(bool verbose, string errorFilename)
+void testStreamError(bool verbose, string errorFilename) @trusted
```

Comment:
Use `readText.representation` instead?

Reply:
No good. Some of the tests use invalid UTF8, and readText reads AND validates.